docs/serving/using-auto-tls.md

```diff
@@ -154,6 +154,9 @@ and which DNS provider validates those requests.
 
 If you choose to use the mode of provisioning certificate per namespace, you need to install `networking-ns-cert` components.
 
+**IMPORTANT:** Provisioning a certificate per namespace only works with DNS-01
+ challenge. This component cannot be used with HTTP-01 challenge.
+
```

Comment:
@ZhiminXiang can you please also add that:

- HTTP-01 does not support provisioning a certificate per namespace

Up above within the list "1.  Using HTTP-01 challenge" (line 30) to make this clear at the time when choosing DNS-01 vs HTTP-01

Reply:
Done